lib/airbrake-ruby/payload_truncator.rb

```diff
@@ -109,11 +109,12 @@ def truncate_string(str)
     # @return [void]
     # @note This method mutates +str+ for speed
```

Comment:
Good call, it makes sense. I guess it's also better to update this note, to mention that it mutates every but frozen strings.